packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -204,11 +204,18 @@ abstract class FlutterDriver {
     await sendCommand(Tap(finder, timeout: timeout));
   }
 
-  /// Waits until [finder] locates the target.
+  /// Waits until [finder] locates the target. [finder] will wait until
```

Comment:
First paragraph should be a single sentence.

Reply:
fixed